jsk_fetch_robot/jsk_fetch.rosinstall.melodic

```diff
@@ -1,71 +1,76 @@
 # This is rosinstall file for melodic PC inside fetch.
 # $ ln -s $(rospack find jsk_fetch_startup)/../jsk_fetch.rosinstall.$ROS_DISTRO $HOME/ros/$ROS_DISTRO/src/.rosinstall
-
 - git:
     local-name: PR2/app_manager
```

Comment:
please add comment such as
```
# In order to avoid issue https://github.com/jsk-ros-pkg/jsk_robot/issues/1665,
# we need to use development branch until PR below are merged
# https://github.com/ZebraDevs/fetch_ros/pull/162
# https://github.com/ZebraDevs/fetch_ros/pull/163
# https://github.com/ZebraDevs/fetch_ros/pull/164
```

Reply:
I update `.rosinstall` and add more comments.